### DISCUSSION TITLE
Questions about the wake merging of multiple wind turbines using FAST.Farm module

### DISCUSSION THREAD

**Discussion by venturi123** [OpenFAST/openfast#1031] [Q&A]
Hi,

I have some questions about the wake merging of multiple wind turbines using `FAST.Farm`.

How to simulate a wind farm with multiple wind turbines located in the wake region of the previous turbine?
Because of the wake effect, the velocity and turbulence intensity at the hub height are variables 
that can not be predicted and set as an Initial or fixed rotor speed at `RotSpeed` entry in the `ELASTODYN INPUT FILE`. 

I wonder if the `FAST.Farm` deal with this problem with the `ServoDyn` module?
If yes, Should I program a new `DISCON.DLL` for a custom wind turbine model
(just like https://github.com/OpenFAST/r-test/blob/7892739f47f312ce014711192fd70253ea40c8e8/glue-codes/fast-farm/5MW_Baseline/ServoData/DISCON/DISCON.F90) to achieve my goals?

Thanks!

**Comment by jjonkman** [comment]
Dear @venturi123,

I'm not sure I fully understand your question.  But I'll just say that you can initialize the rotor speed and blade-pitch angles of each OpenFAST model in FAST.Farm based on the ambient wind speed, just as you would in standalone OpenFAST when not considering wake effects  There is a start-up transient in FAST.Farm whereby the wakes propagate downstream.  So, at the start of each FAST.Farm simulation, all turbines start off unwaked.

Best regards,
